python/pylibcudf/pylibcudf/libcudf/interop.pxd

```diff
@@ -21,15 +21,34 @@ cdef extern from "dlpack/dlpack.h" nogil:
 
 # The Arrow structs are not namespaced.
 cdef extern from "cudf/interop.hpp" nogil:
+    # https://arrow.apache.org/docs/format/CDataInterface.html#structure-definitions
```

Comment:
Rather than adding the entire declaration for completeness, let's just declare the parts that we actually need to access from Cython (which I think is almost none of the schema attributes, but probably almost all of the array attributes?).